gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/accessController/index.ts

```diff
@@ -12,12 +12,17 @@ const getOwner = async (program, state) => {
   return contractState?.owner
 }
 
+const getProposedOwner = async (program, state) => {
```

Comment:
This is basically doing the same as `getOwner`. Let's refactor `getOwner` to `inspectOwnership` and return an `Ownership` type that is:
```js
type Ownership = {
  owner: string
  proposedOwner?: string
}
```
The `inspectOwnership` declaration should go where `makeInspectOwnershipCommand` is.